src/background.js

```diff
@@ -357,8 +357,27 @@ const rollout = {
   },
 };
 
+function studyCheck(study) {
+
+  if (study === undefined) {
+    // throw new Error("No study associated with this extension!");
+    return false;
```

Comment:
I'd definitely go with the `return false` version of this so that we can use a unified version of the add-on for all use cases.